Common/Securities/Future/FutureSymbol.cs

```diff
@@ -0,0 +1,61 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); 
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+using QuantConnect.Logging;
+
+namespace QuantConnect.Securities.Future
+{
+    /// <summary>
+    /// Static class contains common utility methods specific to symbols representing the future contracts
+    /// </summary>
+    public static class FutureSymbol
+    {
+        /// <summary>
+        /// Determine if a given Futures contract is a standard contract.
+        /// </summary>
+        /// <param name="symbol">Future symbol</param>
+        /// <returns>True if symbol expiration matches standard expiration</returns>
+        public static bool IsStandard(Symbol symbol)
+        {
+            var contractExpirationDate = symbol.ID.Date;
+
+            try
+            {
+                // Use our FutureExpiryFunctions to determine standard contracts dates.
+                var expiryFunction = FuturesExpiryFunctions.FuturesExpiryFunction(symbol);
+                var standardExpirationDate = expiryFunction(contractExpirationDate);
+
+                // Return true if the dates match
+                return contractExpirationDate == standardExpirationDate;
```

Comment:
Suggest we compare the `Date` component here, `expiryFunction` might return the hour component, but for example I believe IB won't (see https://github.com/QuantConnect/Lean/blob/master/Brokerages/InteractiveBrokers/InteractiveBrokersBrokerage.cs#L2229). Let's give this case a live test too to be on the safe side👍

Reply:
Good idea! I did run it through a live test with IB and saw the weeklys get filtered out so can confirm it works and there was no issue with the full DateTime comparison. But using the date component is probably a safer bet!

Reply:
Made the change, testing IB again to confirm behaviour!

Reply:
You were correct! The time components of the DateTimes did not match up so it is for the best that we use the `Date` component.